config/vx-mimic3.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Mimic3 Speech Synthesis
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/mimic3-server --voice en_US/m-ailabs_low
```

Comment:
Oh, maybe you are using the server? As best I can tell from a quick read, `/usr/bin/mimic3` just calls into [`mimic3_tts`](https://github.com/MycroftAI/mimic3/blob/master/mimic3_tts/__main__.py#L348) and does not use the server. Is this actually used?

Reply:
I'm fairly certain that the `--remote` option is what makes the `mimic3` client contact the server.